### DISCUSSION TITLE
idk snail ide dev chat :trollface:

### DISCUSSION THREAD

**Discussion by dumorando** [Snail-IDE/snail-ide.github.io#17] [General]
br

**Comment by nmsderp** [comment]
silly

**Reply by dumorando** [reply]
yes!!

**Reply by nmsderp** [reply]
e

**Reply by dumorando** [reply]
anyway im pretty sure making a vercel team costs like $100/month

**Reply by nmsderp** [reply]
💀 ill just ask you if i need to do something with the domain

**Reply by dumorando** [reply]
bet

**Reply by nmsderp** [reply]
anyways we should use status.snail-ide.com for the status page, since i can add domains to the status page

**Reply by dumorando** [reply]
idk if its possible because i set the nameservers of the domain to vercel instead of the dns. which means everything is controlled by vercel

**Reply by nmsderp** [reply]
lets try it

**Reply by dumorando** [reply]
k if it says anything about dns then ye it wont work

**Reply by nmsderp** [reply]
alr tell me when the subdomain is made and stuff

**Reply by dumorando** [reply]
you can just set it. subdomains are already setup like this

**Reply by nmsderp** [reply]
oh nice, but it does this https://status.snail-ide.com/

**Reply by dumorando** [reply]
ye,it wont work. i mean we could just redirect it to hostinger status page :trollface:

**Reply by dumorando** [reply]
thank god,i was wondering what would happen if i did that on a different account XD

**Reply by dumorando** [reply]
like give you permission or just make it myself?
(edit:i would have to make it myself cuz i cant control dns records :bleh:)

**Reply by nmsderp** [reply]
add permission with the steps given by vercel

**Reply by dumorando** [reply]
doesnt work :p

**Reply by dumorando** [reply]
> i would have to make it myself cuz i cant control dns records :bleh:

**Reply by nmsderp** [reply]
its not dns its a vercel app im hosting

**Reply by dumorando** [reply]
1sec

**Reply by dumorando** [reply]
k try now

**Reply by nmsderp** [reply]
https://extcreate.snail-ide.com/ it works

**Reply by dumorando** [reply]
lol turbobuilder is everywhere

**Reply by nmsderp** [reply]
this is a cool feature

**Reply by dumorando** [reply]
?

**Reply by nmsderp** [reply]
the domain thing

**Reply by dumorando** [reply]
o ye, i agree

**Reply by nmsderp** [reply]
its really useful, but i wish it was supported outside of vercel

**Reply by dumorando** [reply]
ye that would be rly nice

**Reply by dumorando** [reply]
also new api is hosted at https://ba4x.pro/experiments/snail-ide so that i can use hostingers storage space

**Reply by nmsderp** [reply]
btw its a 403 error

**Reply by nmsderp** [reply]
might because i just opened the url in my browser

**Reply by dumorando** [reply]
cuz its a directory. didnt put anything on index.php

**Reply by nmsderp** [reply]
also if the frontend is available that would be nice

**Reply by dumorando** [reply]
when i actually get project sharing to work (fyi: its hell trying to make it xd)

**Reply by nmsderp** [reply]
alr if possible put the current php code under a private repo, i might be able to help (with my small knowledge of php lol)

**Reply by dumorando** [reply]
i have WAY too many api keys in there lmao

**Reply by nmsderp** [reply]
wow

**Reply by dumorando** [reply]
actually just 1....yea ill just make it a private repo

**Reply by nmsderp** [reply]
alr

**Reply by dumorando** [reply]
its done

**Reply by nmsderp** [reply]
ty
